src/patient-registration/input/custom-input/relationship-input.component .tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import styles from './../input.css';
+import { SelectInput } from './select-input.component';
+import { getAllRelationshipTypes, getPerson } from '../../patient-registration.resource';
+import { createErrorHandler } from '@openmrs/esm-error-handling';
+import { FieldArray } from 'formik';
+import Autosuggest from 'react-autosuggest';
+//import styles from './autosuggest.css';
+
+interface RelationshipInputProps {
+  setFieldValue(field: string, value: any, shouldValidate?: boolean): void;
+}
+
+export const RelationshipInput: React.FC<RelationshipInputProps> = ({ setFieldValue }) => {
+  const [relationshipTypesOptions, setrelationshipTypesOptions] = useState([]);
```

Comment:
The setter name should be changed to `setRelationshipTypesOptions`.

Reply:
Done